lib/Runner.js

```diff
@@ -1,9 +1,33 @@
 // @flow
 
 const spawn = require('cross-spawn'),
-  firstline = require('firstline'),
+  fs = require('fs-extra'),
+  readline = require('readline'),
   path = require('path');
 
+function getFirstLine(pathToFile /*: string */) /*: Promise<string> */ {
+  return new Promise((resolve, reject) => {
+    const readable = fs.createReadStream(pathToFile);
+    const reader = readline.createInterface({ input: readable });
+    let foundLine = false;
+
+    readable.on('error', (err) => reject(err));
+
+    reader.on('line', (line) => {
+      foundLine = true;
+      reader.close();
+      readable.close();
+      resolve(line);
+    });
+    reader.on('close', () => {
+      if (!foundLine) {
+        reject(new Error(`File ${pathToFile} is empty!`));
+      }
+    });
+    reader.on('error', (err) => reject(err));
+  });
+}
+
```

Comment:
Would it be worth sticking this in a separate file just to keep clutter out of this file?